docs/app/Examples/Button/Labeled.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import {Button} from 'stardust';
+
+export default class extends Component {
+  render() {
+    return (
+      <Button className='labeled'>
+        <div className='ui button'>
+          <i className='heart icon'></i> Like
+        </div>
```

Comment:
Sticking with Stardust components, let use a nested Button here instead of a `div`.

We'll need to make sure the nested button is a `div` for this to work.  This looks like more evidence that we should default the `Button` to render as a `div` and only render as a button in select cases (ie `type='submit'`, etc).

Referencing #46 here as it deals with the type of element rendered by the Button.  Let's add another `// TODO: ...` referencing issue 46 in the code as well.